Sources/Hyperspace/Service/Network/NetworkServiceProtocol.swift

```diff
@@ -38,6 +38,11 @@ public struct NetworkServiceSuccess: Equatable {
 public struct NetworkServiceFailure: Error, Equatable {
     public let error: NetworkServiceError
     public let response: HTTP.Response?
+    
+    public init(error: NetworkServiceError, response: HTTP.Response?) {
```

Comment:
this public init is going to help with unit test for my project where I initialize my custom error with a NetworkServiceFailure
can we get a public init for NetworkServiceSuccess as well?

Reply:
Yes sir.

Reply:
thanks